test/testflows/clickhouse_backup/configs/backup/config.yml

```diff
@@ -72,6 +72,9 @@ general:
   max_file_size: 107374182400
   remote_storage: none
   upload_concurrency: 1
+  restore_database_mapping: 
+    default: default
```

Comment:
we should make another test case instead modify base config
and check 

if you want just cover this case inside ./test/integration/integration_test.go`
i could be easier modify for you

Reply:
I can try it out first

Reply:
your changes works, but broke testflows
look 
https://github.com/AlexAkulov/clickhouse-backup/runs/6466446682?check_suite_focus=true
for details